tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "lib": ["es5", "es6"],
-    "target": "es5",
+    "lib": ["dom", "es5", "es6"],
```

Comment:
Why is the "dom" lib needed? Just asking out of curiousity.

Reply:
@MykleNero thanks for taking the time to look at this! The "dom" was just a quick fix for this error:

```
$ yarn build
yarn run v1.22.17
$ rimraf dist
$ tsc --project ./tsconfig.build.json
node_modules/@types/node/ts4.8/stream.d.ts:842:39 - error TS2304: Cannot find name 'Blob'.

842             static from(src: Stream | Blob | ArrayBuffer | string | Iterable<any> | AsyncIterable<any> | AsyncGeneratorFunction | Promise<any> | Object): Duplex;
                                          ~~~~
Found 1 error in node_modules/@types/node/ts4.8/stream.d.ts:842
```

`Blob` is defined as a global in the browser, but is a member of `buffer` in `node`. It's a little unclear to me what is dependent on `stream`, so adding "dom" to lib is the only workaround I've found so far.